tests/php/includes/rest-api/Controllers/Version3/class-wc-rest-producs-controller-tests.php

```diff
@@ -1,24 +0,0 @@
-<?php
```

Comment:
this files is renamed to fix the typo in file name - `producs` to `products`. More tests are added to git is reading it as deleted file.